scripts/deployer.ts

```diff
@@ -0,0 +1,279 @@
+import 'hardhat-deploy-ethers';
+import * as fs from 'fs';
+import { ContractReceipt, ContractTransaction } from 'ethers';
+import mainnetData from '../mainnet_deployed.json';
+import testnetData from '../testnet_deployed.json';
+import dummyData from '../example_testnet_deployed.json';
+import { getCurrentTimestamp } from 'hardhat/internal/hardhat-network/provider/utils/getCurrentTimestamp';
+import { ethers } from 'hardhat';
+import hre from 'hardhat';
+
+interface DeployedContract {
+  abi: Array<any>; // Improve these types
```

Comment:
This lint warning should be fixed.

Reply:
removed the comment, this any tag is alright